### PR TITLE
Only show open invitations and account registrations for last 3 months.

### PR DESCRIPTION
**Pull request by somexpert**

### CODE REVIEW

**Comment by jason-ellis** [comment]
:+1: Enjoy your :shirt:

**Comment by somexpert** [comment]
ping @ldko, @jason-ellis. And I _will_ enjoy my shirt!

File: invite/views.py [hunk 1]
```diff
@@ -157,12 +159,15 @@ def log_in_user(request):
 @login_required(redirect_field_name=None,
                 login_url=reverse_lazy('invite:login'))
 def index(request):
+    after_date = date.today() - timedelta(weeks=12)
```
Comment (ldko) [inline invite/views.py]: Do you think the timeframe of displayed invitations should be a setting?

Comment (somexpert) [reply invite/views.py]: For some reason that thought did not occur to me. But it makes sense, I'll add that as a setting.

**Comment by jason-ellis** [comment]
:+1:

**Comment by somexpert** [comment]
Fixes #28

**Pull request merged**